index.js

```diff
@@ -49,7 +49,10 @@ function fastifyView (fastify, opts, next) {
 
     const promise = new Promise((resolve, reject) => {
       renderer.apply({
-        getHeader: () => {},
+        getHeader: () => {
```

Comment:
Could you revert this change?
The previous version return `undefined` that is a falsy value so the `if condition` will be false aswell

Reply:
OK I got it.